hack/README.md

```diff
@@ -41,7 +41,7 @@ The standard workflow is performed inside a helper container to normalize the bu
     - No `build-cloner` target exists as the code is written in bash
 - `test`: execute all tests
     - `test-unit`: execute all tests under `./pkg`
-    - `test-functional`: execute all tests under `./test`
+    - `test-functional`: execute all tests under `./tests`
```

Comment:
what do you think about adding a comment here that functional tests are also known as e2e tests?

Reply:
That might be more confusing that illuminating I think.  Also, kubevirt defines e2e as something else i think.

Reply:
ok, didn't realize kubevirt has a def for e2e. Ok, never-mind.